packages/daemon/src/formula-identifier.js

```diff
@@ -0,0 +1,29 @@
+const { quote: q } = assert;
+
+const numberlessFormularIdentifiers = new Set([
+  'pet-store',
+  'host',
+  'endo',
+  'least-authority',
+  'web-page-js',
+]);
```

Comment:
Aside: All of these will have numbers in order to make their formula identifiers unforgeable.